draft-ietf-rtcweb-jsep.xml

```diff
@@ -1165,10 +1165,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         bundle-only. This policy aims to minimize
                         candidate gathering and maximize multiplexing,
                         at the cost of less compatibility with legacy
-                        endpoints. When acting as answerer, if there
-                        if no bundle group in the offer, the
-                        implementation will reject all but the first
-                        m= section.</t>
+                        endpoints. When acting as answerer, the implementation
+                        will accept only the compatible m= sections in the first bundle
```

Comment:
'compatible' is a new concept here; you don't discuss whether the first section is compatible in the non-bundle case.

I guess you could handle this in 3 different ways:
1. "only the first m= section, if compatible"
2. flip the text to discuss rejection, e.g. the impl will reject all bundle groups except the first, or if no bundle groups, all but the first m= section
3. change the text to say 'consider' or some such, e.g. the impl will consider only the m= sections in the first bundle group, or... only the first m= section.

I think I lean toward 2).